app/views/unattended/provisioning_templates/provision/preseed_default.erb

```diff
@@ -125,11 +125,12 @@ d-i apt-setup/local<%= repos %>/key string http://keyserver.ubuntu.com/pks/looku
 <% end -%>
 <% end -%>
 
-<% if host_param('kt_activation_keys') && @host.operatingsystem.name == 'Debian' -%>
-# If we are using Katello for content management, then do not add updates and
-# security repos during the initial installation. Prevents us from defaulting to:
-# d-i apt-setup/services-select multiselect updates,security
+<% if host_param('kt_activation_keys') -%>
```

Comment:
I like the idea of preventing inflationary introduction of new host parameters, but I now believe using `kt_activation_keys` as a proxy for "wants an offline installation" messes up the legitimate use case of having some content come from Katello (for example client software), while everything else (base OS repos) still comes from official upstream mirrors.

As a result I am wondering if it would not be better to use a dedicated parameter:
```suggestion
<% if host_param('offline_installation') -%>
```

Any thoughts? Is mixed Katello and online content a legitimate use case?

Reply:
So if you use Katello for lifecycle management (frozen patchlevels and control over the repositories), you do not want to have upstream-repositories available - especially for the os-repos.
From my point of view it would be strange to control certain repos via Katello, but not the operating-system ones - because you get most of the dependencies there, which may break things

Reply:
Yeah, I'd agree "use katello *and* upstream repos" should be the corner case, not the other way round.
And in that case, people can just re-deploy the upstream repos using the config management of their choice.

Reply:
> Yeah, I'd agree "use katello _and_ upstream repos" should be the corner case, not the other way round.
> And in that case, people can just re-deploy the upstream repos using the config management of their choice.
Or if they do not want to use configuration management for it, they could simply make an edited clone of the template. The good thing is, that everyone can make her/his own edits and use it locally. (You only need to make sure that they still work after updates)

Reply:
Given the general echo I won't pursue this further and consider it resolved.